packages/liveblocks-devtools/src/devtools/icons/tree.tsx

```diff
@@ -246,3 +246,46 @@ export function TrashIcon(props: ComponentProps<"svg">) {
     </svg>
   );
 }
+
+export function TextIcon(props: ComponentProps<"svg">) {
+  return (
+    <svg
+      width="20"
+      height="20"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <rect
+        opacity=".1"
+        width="20"
+        height="20"
+        rx="4.375"
+        fill="currentColor"
+      />
+      <path
+        d="M14 6H5M16 10H5M12.5 14H5"
+        stroke="currentColor"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+}
```

Comment:
We use different icon styles between the dashboard trees and the DevTools ones, I'll update this icon later today to fit the DevTools. (https://www.figma.com/file/WDmXK9fG6aD5OT1AjFVUDr/Liveblocks-%E2%80%93-DevTools?type=design&node-id=0%3A1&mode=design&t=DehwOPjPYvhVbCpt-1)

Reply:
Thank you @marcbouchenoire ❤️